examples/src/main/java/com/mapbox/navigation/examples/core/MapboxCustomStyleActivity.kt

```diff
@@ -97,24 +89,9 @@ class MapboxCustomStyleActivity : AppCompatActivity(), OnMapLongClickListener {
         MapboxSpeedLimitApi(speedLimitFormatter)
     }
 
-    /**
-     * TripProgress: The data in the view is formatted by different formatting implementations.
-     * Below are default formatters using default options but you can use your own formatting
-     * classes.
-     */
```

Comment:
Aren't these comments not useful? Same thing applies to some others below 👇 

cc @cafesilencio

Reply:
Yeah I forgot to remove these comments from this particular activity. @Guardiola31337

Reply:
Wait. This file is deleted. These comments are not there anymore in `examples/src/main/java/com/mapbox/navigation/examples/core/MapboxCustomStyleActivity.java`. 

Don't look at any file under `test-app/src/*` because this folder is deleted.

cc @Guardiola31337

Reply:
For that activity the comment isn't needed IMO.

Reply:
Should we port them over to the `examples/MapboxCustomStyleActivity` then?

Reply:
Or over to `MapboxTripProgressActivity`?

Reply:
These comments are already added to `examples/MapboxTripProgressActivity`